avocado/utils/memory.py

```diff
@@ -41,30 +41,16 @@ def read_from_meminfo(key):
     return int(re.search(r'\d+', meminfo).group(0))
 
 
-def memtotal():
-    """
-    Read ``Memtotal`` from meminfo.
-    """
-    return read_from_meminfo('MemTotal')
-
-
-def freememtotal():
-    """
-    Read ``MemFree`` from meminfo.
-    """
-    return read_from_meminfo('MemFree')
-
-
```

Comment:
+1, the test writer APIs shouldn't be touched unless we go through the proper process of deprecation and removal on a later version.